main.go

```diff
@@ -75,7 +78,16 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	config = ctrl.GetConfigOrDie()
```

Comment:
Seems like the test failed at this line of code: https://github.com/cert-manager/aws-privateca-issuer/actions/runs/10475862575/job/29013543290